.editorconfig

```diff
@@ -18,7 +18,6 @@ max_line_length=100
 [*.sh]
 indent_style=space
 indent_size=2
-max_line_length=100
```

Comment:
Is this intentional?

Reply:
Yes, bash doesn't support multiline well